buildSrc/src/main/java/co/icanteach/projectx/buildsrc/Classpaths.kt

```diff
@@ -1,5 +1,9 @@
 object Classpaths {
     val gradleClasspath = "com.android.tools.build:gradle:${Versions.gradleVersion}"
-    val kotlinGradleClasspath = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlinVersion}"
-    val gradleVersionPlugin = "com.github.ben-manes:gradle-versions-plugin:${Versions.gradleVersionPluginVersion}"
+    val kotlinGradleClasspath =
+        "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlinVersion}"
+    val gradleVersionPlugin =
+        "com.github.ben-manes:gradle-versions-plugin:${Versions.gradleVersionPluginVersion}"
+    val hilt = "com.google.dagger:hilt-android-gradle-plugin:${Versions.hiltVersion}"
+
```

Comment:
Minor: needless empty space

Reply:
Also in: 

- buildSrc/src/main/java/co/icanteach/projectx/buildsrc/Plugins.kt
- buildSrc/src/main/java/co/icanteach/projectx/buildsrc/Versions.kt